Gems/ScriptAutomation/Code/Source/ScriptAutomationSystemComponent.cpp

```diff
@@ -374,4 +387,12 @@ namespace ScriptAutomation
         }
     }
 
-} // namespace ScriptAutomation
+    void ScriptAutomationSystemComponent::OnCatalogLoaded(const char* /*catalogFile*/)
```

Comment:
Either `[[maybe_unused]]` should be used here or the variable name should be deleted.